packages/braid-design-system/src/lib/components/private/Modal/Modal.css.ts

```diff
@@ -34,56 +32,43 @@ const reducedMotion = style({
   },
 });
 
+const rightAnimation = style([
+  reducedMotion,
+  responsiveStyle({
+    mobile: { opacity: 1, transform: 'translateX(110%)' },
+    tablet: { opacity: 0, transform: 'translateX(40px)' },
+  }),
+]);
+
+const leftAnimation = style([
+  reducedMotion,
+  responsiveStyle({
+    mobile: { opacity: 1, transform: 'translateX(-110%)' },
+    tablet: { opacity: 0, transform: 'translateX(-40px)' },
+  }),
+]);
+
+const centerAnimation = style([
+  reducedMotion,
+  {
+    transform: 'scale(.8)',
+  },
+]);
+
 export const entrance = {
-  center: [
-    reducedMotion,
-    style({
-      transform: 'scale(.8)',
-    }),
-  ],
-  right: [
-    reducedMotion,
-    style(
-      responsiveStyle({
-        mobile: { opacity: 1, transform: 'translateX(110%)' },
-        tablet: { opacity: 0, transform: 'translateX(40px)' },
-      }),
-    ),
-  ],
-  left: [
-    reducedMotion,
-    style(
-      responsiveStyle({
-        mobile: { opacity: 1, transform: 'translateX(-110%)' },
-        tablet: { opacity: 0, transform: 'translateX(-40px)' },
-      }),
-    ),
-  ],
+  center: centerAnimation,
+  right: rightAnimation,
+  left: leftAnimation,
 };
 
 export const exit = {
-  right: [
-    reducedMotion,
-    style(
-      responsiveStyle({
-        mobile: { opacity: 1, transform: 'translateX(110%)' },
-        tablet: { opacity: 0, transform: 'translateX(10px)' },
-      }),
-    ),
-  ],
-  left: [
-    reducedMotion,
-    style(
-      responsiveStyle({
-        mobile: { opacity: 1, transform: 'translateX(-110%)' },
-        tablet: { opacity: 0, transform: 'translateX(-10px)' },
-      }),
-    ),
-  ],
+  center: centerAnimation,
```

Comment:
Might be worth adding a changeset for the introduction of the `Dialog` exit animation.